src/crypto/local.js

```diff
@@ -141,8 +198,11 @@ function compressPoint(P) {
  */
 function decompressPoint(bytes) {
     const yTag = bytes[0];
+    const expLength = Math.ceil(sjcl.bitArray.bitLength(CURVE.r.toBits())/8)+1;
```

Comment:
if ``sjcl.bitArray.bitLength(CURVE.r.toBits()) == k*8`` then this will result in (k+1) bytes. Seems to me check at line 204 will fail

Reply:
ah,, no - ignore. that's OK